Linux.Vulnerable.Dockerfile

```diff
@@ -20,6 +20,12 @@ RUN /dotnet-retire/assert-cmd.sh 3 "dotnet retire --loglevel=debug"
 RUN /dotnet-retire/assert-cmd.sh 3 "dotnet retire"
 RUN /dotnet-retire/assert-cmd.sh 0 "dotnet retire --ignore-failures"
 
+WORKDIR /OtherVulnerableApp
+COPY SampleProjects/OtherVulnerableApp/OtherVulnerableApp.csproj ./
+RUN /dotnet-retire/assert-cmd.sh 3 "dotnet retire --loglevel=debug"
+RUN /dotnet-retire/assert-cmd.sh 3 "dotnet retire"
+RUN /dotnet-retire/assert-cmd.sh 0 "dotnet retire --ignore-failures"
+
```

Comment:
It may not actually be needed now... I originally made it because I wanted an example where I had the same error coming from two projects under analysis. The SLN did not contain the third project that was already present, which I think was the root of the issue I was trying to sort out, and ended up fixing along the way.

We can probably kill the 'OtherVulnerableApp' project.